lib/rebaser-server/src/rebase.rs

```diff
@@ -502,17 +510,32 @@ pub(crate) async fn send_updates_to_edda_legacy_snapshot(
     Ok(())
 }
 
-pub(crate) async fn evict_unused_snapshots(
+pub(crate) async fn evict_unused_snapshot(
```

Comment:
Well, the singular form of this function name is a lot more accurate than the plural form